zio-schema-json/shared/src/main/scala/zio/schema/codec/JsonCodec.scala

```diff
@@ -448,14 +450,12 @@ object JsonCodec {
                   var first = true
                   values.foreach {
                     case (key, value) =>
-                      if (first)
-                        first = false
+                      if (first) first = false
```

Comment:
Please avoid unrelated format changes

Reply:
It is my attempt to make this peace of code looks similar though different copies in `JsonCodec.scala`